apps/aechannel/src/aesc_state_cache.erl

```diff
@@ -79,46 +70,50 @@
 -type ch_cache_id() :: { aeser_id:val()
                        , aeser_id:val() | atom()}.
 -type opts() :: map().
--record(ch_cache, { cache_id    :: ch_cache_id() | undefined
-                  , state       :: aesc_offchain_state:state() | atom() | undefined
-                  , opts = #{}  :: map() | '_'
-                  , salt        :: binary() | atom()
-                  , session_key :: binary() | atom()
+-record(ch_cache, { cache_id       :: ch_cache_id() | undefined
+                  , state          :: aesc_offchain_state:state() | atom() | undefined
+                  , opts = #{}     :: map() | '_'
+                  , fsm_id_wrapper :: aesc_fsm_id:wrapper() | atom()
                   }).
 
--define(VSN_pch, 1).
--record(pch, {id, pubkeys = [], state}).
--define(VSN_pch_encrypted_cache, 2).
+-define(VSN_pch_encrypted_cache, 1).
 -record(pch_encrypted_cache, { cache_id        :: ch_cache_id()
-                             , salt            :: binary() | atom()
                              , nonce           :: binary() | atom()
                              , encrypted_state :: binary() | atom()
                              , opts = #{}      :: opts() | '_'         % unencrypted
                              }).
 
 -define(SERVER, ?MODULE).
 -define(TAB, aesc_state_cache_ch).
--define(PTAB, aesc_state_cache).
--define(CACHE_DEFAULT_PASSWORD, "correct horse battery staple").
-
--spec new(aeser_id:val(), aeser_id:val(), aesc_offchain_state:state(), opts()) -> ok | {error, any()}.
-new(ChId, Pubkey, State, Opts) ->
-    new(ChId, Pubkey, State, Opts, ?CACHE_DEFAULT_PASSWORD).
-
--spec new(aeser_id:val(), aeser_id:val(), aesc_offchain_state:state(), opts(), string()) -> ok | {error, any()}.
-new(ChId, PubKey, State, Opts, Password) ->
-    gen_server:call(?SERVER, {new, ChId, PubKey, self(), State, Opts, unicode:characters_to_binary(Password)}).
-
--spec reestablish(aeser_id:val(), aeser_id:val()) -> {ok, aesc_offchain_state:state(), opts()} | {error, atom()}.
-reestablish(ChId, Pubkey) ->
-    reestablish(ChId, Pubkey, ?CACHE_DEFAULT_PASSWORD).
-
--spec reestablish(aeser_id:val(), aeser_id:val(), string()) -> {ok, aesc_offchain_state:state(), opts()} | {error, atom()}.
-reestablish(ChId, PubKey, Password) ->
-    gen_server:call(?SERVER, {reestablish, ChId, PubKey, self(), unicode:characters_to_binary(Password)}).
-
-change_state_password(ChId, Pubkey, Password) ->
-    gen_server:call(?SERVER, {change_state_password, ChId, Pubkey, unicode:characters_to_binary(Password)}).
+%% A new table name is needed in case the user want's to downgrade to a pre-lima release
```

Comment:
Also, `aehttp_sc_SUITE.erl` still has a test, `sc_ws_basic_open_close_no_password()`, which I believe amounts to a simple `basic_open_close`, since the `no_password` no longer means anything.